### DISCUSSION TITLE
Window version

### DISCUSSION THREAD

**Discussion by Ebm2003NYC** [NickvisionApps/Denaro#747] [General]
Hi,

I like your work, and I would like to know if you will add a Window version of Denaro? Currently I'm using Home bank, but I'd like to use new software, like Denaro!

Thank you

**Comment by nlogozzo** [comment]
Yes it is planned.

**Reply by Ebm2003NYC** [reply]
Sounds good, thanks for your work!

**Comment by nlogozzo** [comment]
Track progress in #784
